indico/htdocs/js/indico/modules/designer.js

```diff
@@ -471,17 +474,27 @@
             backside_template_id: backsideTemplateID
         };
 
-        $.ajax({
-            url: location.pathname,
-            data: JSON.stringify(templateData),
-            contentType: 'application/json',
-            method: 'POST',
-            complete: IndicoUI.Dialogs.Util.progress(),
-            error: handleAjaxError,
-            success: function(data) {
-                handleFlashes(data, true);
-            }
-        });
+        var confirmed = showConfirmationDialog ? confirmPrompt(
+            $T.gettext("Are you sure you want to change the dimensions of this template?") +
+            $T.gettext("The back side of the templates that use this one as the back side will be removed."),
+            $T.gettext("Disassociate back sides")
+        ) : $.Deferred().resolve();
+
+        confirmed.then(saveData);
```

Comment:
You could for an anonymous function here since you only use it once now:

    confirmed.then(function() { ... })